dangerfile.js

```diff
@@ -1,55 +1,99 @@
-const count = (string, substring) => (string.match(new RegExp(substring, 'g')) || []).length;
+const {
+    message,
+    danger,
+    warn,
+    fail
+} = require('danger');
 
 const {
-    github: {
-        pr: {
-            body,
-            base: {
-                ref,
-            },
-        },
-    },
     git: {
-        modified_files,
-        diffForFile,
-    },
+        modified_files: modifiedFiles,
+        diffForFile
+    }
 } = danger;
 
-function noPRDescription(callback = message) {
-    if (!body || body.trim().length < 2) {
-        callback('Please add a description to the pull request.');
-    }
+// Fails if a description section is missing.
+const includesDescription =
+    !danger.github.pr.body ||
+    !danger.github.pr.body.includes('## Description');
+if (!includesDescription) {
+    const title = 'Missing Description?';
+    const idea =`
+        Please add a description. To do so, add a "## Description" section to your PR description.
+        This is a good place to explain all your intentions.
```

Comment:
Instead of `This is a good place to explain all your intentions.`, do:
`This is a good place to explain your intentions and show off screenshots of either new features or areas that have changed.`